### PR TITLE
Solve the issue #2: Failed to login

### PR DESCRIPTION
**Pull request by seokbeomKim**
By using 'Ghost' instead of spidermonkey, issue related to cookie has been solved.

### CODE REVIEW

File: ndrive/client.py [hunk 1]
```diff
@@ -57,12 +65,51 @@ class Ndrive(object):
     session = requests.session()
 
     def __init__(self, debug = False, NID_AUT = None, NID_SES= None):
+        sys.stdout.write('ndrive init')
```
Comment (carpedm20) [inline ndrive/client.py]: This is not necessary.

File: ndrive/cmdline.py [hunk 2]
```diff
@@ -24,6 +23,7 @@ def __init__(self, id, passwd):
         cmd.Cmd.__init__(self)
         self.n = Ndrive()
 
+        print 'id'
```
Comment (carpedm20) [inline ndrive/cmdline.py]: This is not necessary.

File: ndrive/client.py [hunk 3]
```diff
@@ -75,19 +122,21 @@ def login(self, user_id, password, svctype = "Android NDrive App ver", auth = 0)
...
-
+        
         try:
-            cookie = getCookie(user_id, password)
+            cookie = self.getCookie()
         except:
             print "[*] Error getCookie: failed"
             return False
 
+        print "cookie['nid_aut'] = " + cookie["NID_AUT"]
```
Comment (carpedm20) [inline ndrive/client.py]: This is not necessary.

**Comment by carpedm20** [comment]
Thank you for your big contribution but I can't accept your pull request because:

- you should not add build directory or TAGS files in pull request
- there is unnecessary print codes
- you need to update [setup.py](https://github.com/carpedm20/ndrive/blob/master/setup.py) to solve dependency problem

Please fix these issues and give me a new pull reuqest.

**Comment by seokbeomKim** [comment]
Thank you for spending you time and giving me the detail feedback.
I appreciate to give me those feedback since it was the first time that I tried to do pull request.

**Comment by carpedm20** [comment]
Installing Ghost is somewhat too heavy solution and #4 solved this problem by implementing login code only with Python.

**Pull request closed without merging**